frontend/src/static/app/pages/client/alter-check/alter-check.ts

```diff
@@ -0,0 +1,280 @@
+    this.update()
+  },
+  /**
+   * OSごとのファイルマネージャーでalterディレクトリを開きます
+   */
+  onclickOpenAlterDir() {
+    api.openAlterDir(this.props.projectName)
+  },
+  /**
+   * AlterCheckの実行結果htmlを表示します
+   */
+  onclickOpenAlterCheckResultHTML() {
+    window.open('api/document/' + this.props.projectName + '/altercheckresulthtml/')
+  },
+  /**
+   * AlterCheckをAPI経由で実行します
+   * confirmを許可した場合のみ実行されます
+   */
+  onclickAlterCheckTask() {
+    this.suConfirm('Are you sure to execute AlterCheck task?').then(() => {
+      this.update({ executeStatus: 'Executing', executeResultMessage: 'Executing...' })
+      api
+        .task(this.props.projectName, 'alterCheck')
+        .then((data) => {
+          const executeResultMessage = data.success ? 'Success' : 'Failure'
+          this.updateContents({ executeStatus: 'Completed', executeResultMessage })
+        })
+        .catch(() => {
+          // APIリクエストに失敗した際の情報も反映するため更新（一緒に実行モーダルは閉じる）
+          this.updateContents({ executeStatus: 'None' })
+        })
+    })
+  },
+  // ===================================================================================
+  //                                                                             Private
+  //                                                                             =======
+  /**
+   * 画面全体を更新します
+   * @param additionalState 一緒に更新したいstate. 指定しなくてもOK
+   */
+  updateContents(additionalState?: Partial<State>) {
+    api.alter(this.props.projectName).then((result) => {
+      api.clientPropbase(this.props.projectName).then(async (client) => {
+        const editingSqls = result.editingFiles.map((file) => ({
+          fileName: file.fileName,
+          content: Prism.highlight(file.content.trim(), Prism.languages.sql, 'sql'),
+          show: false,
+        }))
+        const unreleasedDir = this.prepareUnreleased(result.unreleasedDir)
+        const checkedZip = this.prepareChecked(result.checkedZip, unreleasedDir)
+        const latestResult = await this.prepareLatestFailureResult(result.ngMarkFile)
+        this.update({
+          hasAlterCheckResultHtml: client.hasAlterCheckResultHtml,
+          editingSqls,
+          unreleasedDir,
+          checkedZip,
+          latestResult,
+          ...additionalState,
+        })
+      })
+    })
+  },
+  /**
+   * 未リリースチェック済みのAlterDDLを用意します（sqlファイルのみ）
+   * sqlファイルはシンタックスハイライトされた状態でセットします
+   * @param {AlterSQLResultUnreleasedDirPart | undefined} unreleasedDir APIで取得した未リリースディレクトリ情報 (Nullable)
+   * @return {AlterDir} 未リリースAlterディレクトリのState情報
+   */
+  prepareUnreleased(unreleasedDir: AlterSQLResultUnreleasedDirPart | undefined): AlterDir {
+    if (!unreleasedDir) {
+      return { checkedFiles: [] }
+    }
+    return {
+      checkedFiles: unreleasedDir.checkedFiles
+        .filter((file) => file.fileName.includes('.sql'))
+        .map((file) => ({
+          fileName: file.fileName,
+          content: Prism.highlight('\n' + file.content.trim(), Prism.languages.sql, 'sql'),
```

Comment:
[Q]
同様のコメントが別箇所にあったので記載漏れですかね？
```suggestion
          // sqlファイル表示時に余白を用意するために改行(\n)を予め入れておく（多分そういう目的）
          content: Prism.highlight('\n' + file.content.trim(), Prism.languages.sql, 'sql'),
```

Reply:
[IMO]
同じ処理があるので、mapの中身は切り出してもよかったかも 🤔